i2c/CMakeLists.txt

```diff
@@ -2,4 +2,5 @@ if (NOT PICO_NO_HARDWARE)
     add_subdirectory(bus_scan)
     add_subdirectory(lcd_1602_i2c)
     add_subdirectory(mpu6050_i2c)
+    add_subdirectory(bmp280_i2c)
```

Comment:
we tend to keep subdirectories in alphabetical order (perhaps hard to infer from these 3 data points!)

Reply:
Ah, yes. Fixed.